src/extension/webviews/query_page/App.tsx

```diff
@@ -139,6 +141,9 @@ export const App: React.FC = () => {
               const data = result.data;
               setJSON(JSON.stringify(data.toObject(), null, 2));
               setSQL(result.sql);
+              if (message.stats) {
+                setStats(getStats(message.stats));
+              }
```

Comment:
Sorry, missed this first pass - this should probably get cleared out between runs, like warning and error. Other than that looks GTG.